dev/NavigationView/NavigationView.cpp

```diff
@@ -4040,18 +4059,34 @@ void NavigationView::OnIsPaneOpenChanged()
 
     if (SharedHelpers::IsThemeShadowAvailable())
```

Comment:
>IsThemeShadowAvailable [](start = 23, length = 22)

Should we introduce an IsDropShadowAvailable shared helper method so we can store the result in a static bool and don't have to query the platform version all the time?

Reply:
That's a great idea, and can be reusable across other new fixes involving shadows. I think that can be done with another PR.

Reply:
Created issue #4388 for tracking